src/main/java/data/Vendor.java

```diff
@@ -33,6 +35,19 @@ public Vendor(String vendorID, String legacyVendorID, int nextGenVendorID) {
     accountList = new ArrayList<Account>();
   }
 
+  public Vendor(HttpServletRequest request, String vendorID, int numAccounts) {
+    // where does the account ID come from?
```

Comment:
It would depend on which account ID we use in this case... I would say we could use legacy account ID